src/main/resources/resource/WebGui/app/service/views/IntroGui.html

```diff
@@ -104,7 +104,6 @@
                 </tr>
                 <tr>
                     <td>
-                        <img width="600" src="http://myrobotlab.org/sites/default/files/users/user3images/Serial_1.png"/>
```

Comment:
How come this was removed?

Reply:
because it doesn't exist .. it returns with a 404